app/routes/administrative-units/administrative-unit/core-data/index.js

```diff
@@ -90,14 +91,29 @@ export default class AdministrativeUnitsAdministrativeUnitCoreDataIndexRoute ext
     }
 
     let kboContacts = A();
-
     let kboAdministrativeUnit = await administrativeUnit.kboAdministrativeUnit;
+
+    if (!kboAdministrativeUnit) {
+      const identifiers = await administrativeUnit.identifiers;
+      const kboIdentifier = identifiers.find((id) => {
+        return id.idName === ID_NAME.KBO;
+      });
+      const structuredIdKBO = await kboIdentifier.structuredIdentifier;
+      console.log(structuredIdKBO);
```

Comment:
**Nitpick:** Do not forget to remove this and any other `console.log` statements before merging.